docs/sources/features/datasources/loki.md

```diff
@@ -134,6 +134,35 @@ Note that Live Tailing relies on two Websocket connections: one between the brow
 ```
 ProxyPassMatch "^/(api/datasources/proxy/\d+/loki/api/v1/tail)" "ws://127.0.0.1:3000/$1"
 ```
+Example below shows basic NGINX proxy configuration. Example assumes that Grafana server is available at `http://localhost:3000/`, Loki server is running locally without proxy and your external site uses HTTPS. If you also host Loki behind nginx proxy, you might want to repeat following configuration for Lokis as well.
+
+In `http` section of NGINX add following map definition:
+```
+  map $http_upgrade $connection_upgrade {
+    default upgrade;
+    '' close;
+  }
+```
+In your `server` section add following configuration:
```

Comment:
```suggestion
In your `server` section, add following configuration:
```

Reply:
Are you sure about the comma? What about `the` before `following`

Reply:
Good point. Added.